src/containers/FaqContainer/index.jsx

```diff
@@ -1,32 +1,24 @@
 import React from 'react'
 import { Accordion } from '../../components'
 import data from '../../json/_faq.json'
-import { Plus } from '../../components/Icon/'
 
 const FaqContainer = () => {
 	return (
 		<Accordion>
 			<Accordion.Heading>Frequently Asked Questions</Accordion.Heading>
 			<Accordion.Inner>
-				{
-					data.map( ( acc, i ) => (
-						<Accordion.Frame key={ i }>

```

Comment:
for the key, use `import { v4 as uuid } from 'uuid'`

Reply:
uuid throws a whole bunch of errrors on console